Automattic/jetpack
# Social: Fix connection refresh logic

Fixes https://github.com/Automattic/jetpack-reach/issues/451

## Proposed changes:

* Removed the `PublicizeConnectionVerify` component and used the more robust hook for that
* Initially we instantly fire the request, otherwise we debounce it for 2 seconds

### Other information:


## Jetpack product discussion


https://github.com/Automattic/jetpack-reach/issues/451
## Does this pull request change what data or activity we track or use?


No.
## Testing instructions:






* Go to the editor.
* Switch to the jetpack tab, in the network tab you should instantly see `connections?test_connections=1` request.
* Swithc back to the WP tab, and to Jetpack again, it should trigger again without delay.
* Now switch tabs in the browser.
* Going back to the page you should see to same request after 2 seconds
